web/templates/pages/member_page.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}{{ user.fullname }}{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+			{{ user.fullname }} 
```

Comment:
the same comment as [here](https://github.com/CodeCatz/TrackCat/pull/143/files#r24723121)